app/components/Common/CodeBlock.tsx

```diff
@@ -0,0 +1,46 @@
+import { SubmitButton } from '../form/SubmitButton'
+import { ContentContainer } from './ContentContainer'
+import { Title } from './Title'
+
+const CodeBlock = ({
+  codeString,
+  title,
+  buttonTitle,
+  onClick,
+  children
+}: {
+  codeString: string
+  title: string
+  buttonTitle: string
+  onClick?: () => void
+  children?: React.ReactNode
+}) => {
+  return (
+    <ContentContainer>
+      <div className="px-2 sm:px-5 pb-2 sm:pb-4">
+        <div className="flex justify-between pb-2 sm:pb-4">
+          <Title title={title} />
+          <SubmitButton
+            title={buttonTitle}
+            type="button"
+            onClick={async () => {
+              if (!window.isSecureContext) {
```

Comment:
`window` shouldn't be referenced as a good practice in Remix (or usually any React framework) due to the server/client rendering they do with code.

Wrap any `window` refernces with document defined checks:

https://remix.run/docs/en/v1/guides/constraints#document-guard

Reply:
fixed